protected/humhub/modules/user/messages/ru/base.php

```diff
@@ -30,94 +30,95 @@
+  'Pending Requests' => 'Запросы ожидающие рассмотрения',
   'People' => 'Люди',
-  'Permission' => 'Разрешение',
+  'Permission' => 'Права и разрешения',
   'Profile' => 'Профиль',
-  'Profile posts only' => 'Только посты пользователя',
-  'Register now and participate!' => 'Зарегистрируйтесь сейчас',
+  'Profile posts only' => 'Только записи пользователя',
+  'Register now and participate!' => 'Зарегистрируйтесь прямо сейчас и участвуйте!',
   'Search...' => 'Поиск...',
   'Security' => 'Безопасность',
   'Settings' => 'Настройки',
-  'Show At Registration' => 'Показать при Регистрации',
-  'Show all content' => 'Показать все содержимое',
+  'Show At Registration' => 'Показать при регистрации',
+  'Show all content' => 'Показать всё содержимое',
   'Sign in' => 'Войти',
-  'Sign in / up' => 'Войти / Присоединиться',
+  'Sign in / up' => 'Войти / Зарегистрироваться',
   'Sign up' => 'Зарегистрироваться',
   'Sign up now' => 'Зарегистрироваться сейчас',
   'Sort order' => 'Порядок сортировки',
   'Source' => 'Источник',
   'Status' => 'Статус',
   'Tags' => 'Теги',
-  'The Group Manager can approve pending registrations of this group.' => 'Менеджер группы может одобрить оставшиеся регистрации этой группы.',
-  'The secure link has been copied in your clipboard!' => 'Защищенная ссылка скопирована в буфер обмена!',
+  'The Group Manager can approve pending registrations of this group.' => 'Менеджер группы может одобрить оставшиеся регистрации в этой группе.',
+  'The secure link has been copied in your clipboard!' => 'Защищённая ссылка была скопирована в ваш буфер обмена!',
   'The user has been blocked for you.' => 'Пользователь был заблокирован для вас.',
   'The user has been unblocked for you.' => 'Пользователь был разблокирован для вас.',
-  'Try other keywords or remove filters.' => 'Попробуйте другие ключевые слова или удалите фильтры.',
+  'Try other keywords or remove filters.' => 'Попробуйте другие ключевые слова или уберите фильтры.',
   'Unblock user' => 'Разблокировать пользователя',
-  'Unknown user status!' => 'Неизвестный статус пользователья!',
-  'Unsuccessful login attempts since last login: {failedLoginAttemptsCount}' => 'Ошибок ввода пароля после последнего входа: {failedLoginAttemptsCount}',
-  'Updated at' => 'Обновлено',
+  'Unknown user status!' => 'Неизвестный статус пользователя!',
+  'Unsuccessful login attempts since last login: {failedLoginAttemptsCount}' => 'Неудачные попытки входа в систему с момента последнего входа в систему: {failedLoginAttemptsCount}',
+  'Updated at' => 'Обновлено в',
   'Updated by' => 'Обновлено пользователем',
   'User' => 'Пользователь',
   'User Group' => 'Группа пользователей',
   'User has been invited.' => 'Пользователь был приглашён.',
-  'Username' => 'Имя пользователя',
-  'Username contains invalid characters.' => 'Имя пользователя содержит недопустимые символы.',
-  'View your about page' => 'Просмотр Вашей страницы «Обо мне»',
+  'Username' => 'Имя пользователя (юзернейм)',
```

Comment:
We can change it to `Логин`, but for russian users, who used HumHub previously it could be undesirable change of term  (for the shorten view only).

So, idk...

Reply:
@hide-me I've never seen a form with fields "Юзернейм/Пароль".
I've seen only "Имя пользователя/пароль" или "Логин/Пароль".
The "Имя пользователя (юзернейм)" is so long, it may be breaked to new line when it will be unexpected.

Reply:
Ok, may vwe keep this string as `Имя пользователя` without `(юзернейм)`? It would be a good compromise?